.gitignore

```diff
@@ -35,3 +35,4 @@ plugins/uefi-dbx/tests/
 .buildconfig
 .ossfuzz
 *.rej
+/snap
```

Comment:
Ohh?

Reply:
When you build the snap you need to put the YAML file in that directory, but we should never commit that directory

Reply:
Snap is quite opinionated it seems :)